lib/adapters/default.js

```diff
@@ -58,6 +58,15 @@ function requestProxy(options = {}) {
         try {
           const res = arguments[0];
           const resObj = mapInRes(res, reqObj, startedAt, reqId, opts);
+          const shouldObfuscatePath =
```

Comment:
this could be extracted as a function, instead of having it written two times in the same file

Reply:
ta megala pneumata sunantontai 😛